spec/index.html

```diff
@@ -985,6 +992,36 @@ <h6 class="defcap">RDF Semantic Conditions</h6>
             <em>IEXT(I(<code>rdf:type</code>))</em> if and only if <em>x</em> is in the 
             <a>value space</a> of <em>I(aaa)</em>.
           </p>
+          <span class="space"></span>
+          
+          <p id="rdfsemcond4"><span style="background-color: Cyan;color: Black">
+            if <em>[I+A](E) = TRUE</em> and <em>E.s</em> is a triple term, then
+            <em>&lt;[I+A](E.s), I(<code>rdf:TripleTerm</code>)&gt;</em> is in <em>IEXT(I(<code>rdf:type</code>))</em>.
+          </span></p>
+          <span class="space"></span>
+      
+          <p id="rdfsemcond5"><span style="background-color: Cyan;color: Black">
+            if <em>[I+A](E) = TRUE</em> and <em>E.o</em> is a triple term, then
+            <em>&lt;[I+A](E.o), I(<code>rdf:TripleTerm</code>)&gt;</em> is in <em>IEXT(I(<code>rdf:type</code>))</em>.
+          </span></p>
+          <span class="space"></span>
+       
+          <p id="rdfsemcond6"><span style="background-color: Cyan;color: Black">
+            if <em>[I+A](E) = TRUE</em> and 
+            <em>&lt;[I+A](E.o), [I+A](rdf:TripleTerm)&gt;</em> is in <em>IEXT(I(<code>rdf:type</code>))</em> and
+            <em>E.p ≠ <code>rdf:type</code></em>, then
+            <em>&lt;[I+A](E.p), I(<code>rdf:ReificationProperty</code>)&gt;</em> is in <em>IEXT(I(<code>rdf:type</code>))</em>.
+          </span></p>
+          <span class="space"></span>
+       
+          <p id="rdfsemcond7"><span style="background-color: Cyan;color: Black">
+            if <em>[I+A](E) = TRUE</em> and
+            <em>&lt;[I+A](E.s), [I+A](rdf:TripleTerm)&gt;</em> is in <em>IEXT(I(<code>rdf:type</code>))</em>
+            and <em>E.p ≠ <code>rdf:type</code></em>, then
```

Comment:
I understand that, without this exception, `rdf:type` would automatically become an InverseReificationProperty (because of the rules above).

Yet, would that be so bad?

Reply:
Yes, it would be bad for several reasons, not least, as you say, because "ReificationProperty was proposed as a mere "flag" to allow people to detect errors", while here there is no error.

Reply:
if the user writes something like x rdf:type <<( s p o )>>, then flagging rdf:type as a reification property IS NOT appropriate, since rdf:type is meant to have a specific meaning (made explicit in RDFS), and the user is expected to respect that intended meaning, and it is not our job to emphasizes that they are making a mistake.

Reply:
I see that my comment below is already made by PA. I also do not see the problem. So, we would want that reification properties have ONLY triple terms in object position or what is our goal here?